src/loaders/common/loader_common_names.py

```diff
@@ -129,6 +129,13 @@
                    'APPDEV': 'https://appdev.kbase.us/services/',
                    'PROD': 'https://kbase.us/services/'}
 
+# containers.conf path
+CONTAINERS_CONF_PATH = ".config/containers/containers.conf"
```

Comment:
```suggestion
CONTAINERS_CONF_PATH = "~/config/containers/containers.conf"
```

Reply:
ah, you mean CONTAINERS_CONF_PATH = "~/.config/containers/containers.conf"?

Reply:
Right, whoops

Reply:
Updated!

Reply:
:+1: